apps/storybook-react/stories/Dialog.stories.tsx

```diff
@@ -0,0 +1,178 @@
+import React, { Fragment, useState } from 'react'
+import {
+  Dialog,
+  DialogProps,
+  Button,
+  Scrim,
+  Typography,
+} from '@equinor/eds-core-react'
+import styled from 'styled-components'
+import { Story, Meta } from '@storybook/react/types-6-0'
+
+const { Actions, Title, CustomContent } = Dialog
+
+export default {
+  title: 'Components/Dialog',
+  component: Dialog,
+  subcomponents: { Actions, Title, CustomContent },
+  parameters: {
+    docs: {
+      description: {
+        component: `A basic dialog component`,
+      },
+    },
+  },
+} as Meta
+
+const Body = styled.div`
+  height: 200px;
+  background: #ebebeb;
+  display: grid;
+  grid-template-rows: 1fr auto 1fr;
+  padding: 32px;
+  grid-gap: 32px;
+  position: relative;
+`
+
+const TempButtonWrapper = styled.div`
+  display: grid;
+  column-gap: 16px;
+  grid-template-columns: repeat(2, fit-content(100%));
+  justify-content: end;
+  justify-self: end;
+`
+
+const Placeholder = styled.div`
+  background: rgba(255, 146, 0, 0.15);
+  border: 1px dashed #ff9200;
+  box-sizing: border-box;
+  border-radius: 4px;
+  padding: 8px;
+  width: 100%;
+  display: inline-block;
+`
+
+const TITLE_CHOICES = {
+  none: null,
+  text: 'Title',
+}
+
+const CUSTOM_CONTENT_CHOICES = {
+  none: null,
+  empty: <Placeholder>Custom content</Placeholder>,
+  emptyLarge: (
+    <Placeholder>
+      Custom content in a larger placeholder. No actions, only ESC or timedelay?
+      Test testestsetsest
+    </Placeholder>
+  ),
+  description: (
+    <Typography variant="body_short">Small description here.</Typography>
+  ),
+  scroll: (
+    <Fragment>
+      <Typography variant="body_short">
+        Lorem ipsum dolor sit amet consecteur dit lot. Lorem ipsum dolor sit
+        amet consecteur dit lot. Lorem ipsum dolor sit amet consecteur dit lot.{' '}
+      </Typography>
+      <Typography variant="body_short">
+        Lorem ipsum dolor sit amet consecteur dit lot. Lorem ipsum dolor sit
+        amet consecteur dit lot. Lorem ipsum dolor sit amet consecteur dit lot.{' '}
+      </Typography>
+    </Fragment>
+  ),
+}
+
+const ACTION_CHOICES = {
+  none: null,
+  buttons: (
+    <TempButtonWrapper>
+      <Button>OK</Button>
+      <Button variant="ghost">Cancel</Button>
+    </TempButtonWrapper>
+  ),
+}
+
+export const Default: Story<DialogProps> = () => (
+  <Dialog>
+    <Title>{TITLE_CHOICES['text']}</Title>
+    <CustomContent>{CUSTOM_CONTENT_CHOICES['description']}</CustomContent>
+    <Actions>
+      <TempButtonWrapper>
+        <Button>Cancel</Button>
+        <Button>OK</Button>
```

Comment:
Cancel left, OK right… 🤔 I think this has gone unnoticed in the original story…

Reply:
![image](https://user-images.githubusercontent.com/2081882/98210577-0e688c00-1f41-11eb-8c56-9a2605b3219b.png)

Reply:
So affirmative actions always on the left, destructive actions on the right.

Reply:
Is it not the other way around? 🙈

Reply:
This is from Figma, but I don’t know _who_ made that in Figma. Maybe there’s something in the Storefront…

Reply:
Can confirm that this is correct